packages/next-app/src/app/internalmanager/(components)/TokenTable.tsx

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable no-console */
```

Comment:
`#curious` doesn't this need `use client` ? 🤔

Reply:
Almost sure that if the father component has `use client` the children don't need to

Reply:
right but that's not the ultimate goal right? We want only the components that actually need client's APIs to be marked with `use client`: https://beta.nextjs.org/docs/rendering/server-and-client-components#when-to-use-server-vs-client-components

Reply:
oh yes, already fixed with `TokenTable` component being directly on `page.tsx` now =)